service/controller/key/cilium.go

```diff
@@ -0,0 +1,12 @@
+package key
+
+import (
+	"github.com/giantswarm/k8smetadata/pkg/annotation"
+	apiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+func ForceDisableCiliumKubeProxyReplacement(cluster apiv1beta1.Cluster) bool {
+	v, found := cluster.Annotations[annotation.CiliumForceDisableKubeProxyAnnotation]
+
+	return found && v == "true"
```

Comment:
Do we wanna check the value or would it be enough to just look if the annotation is set?

Reply:
I would be confused if I see the annotation set to "false" but the behaviour still happening.
I know elsewhere we only check the presence of the annotation, but I think it's best to be explicit and check the value as well.

Reply:
Makes sense 👍🏻